tests/mock_custom/nrf_modem_at/Kconfig

```diff
@@ -0,0 +1,36 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menuconfig MOCK_CUSTOM_NRF_MODEM_AT
+	bool "nRF91 LTE Link control library"
+
+if MOCK_CUSTOM_NRF_MODEM_AT
+
+config MOCK_CUSTOM_NRF_MODEM_AT_SCANF_CALL_COUNT
+	int "Maximum number nrf_modem_at_scanf calls"
+	default 10
+	help
+	  Maximum number nrf_modem_at_scanf calls that can be added before
+	  resetting expected calls. Normally resetting is done before each test case.
+
+config MOCK_CUSTOM_NRF_MODEM_AT_SCANF_VARGS_COUNT
+	int "Maximum number nrf_modem_at_scanf variable arguments"
+	default 10
+	help
+	  Maximum number nrf_modem_at_scanf calls that can be added before
+	  resetting expected calls.
+
+config MOCK_CUSTOM_NRF_MODEM_AT_SCANF_VARGS_STR_SIZE
+	int "Maximum size of a string variable in nrf_modem_at_scanf variable arguments"
+	default 4
```

Comment:
Yep, we do have memory. On the other hand, strings are not needed in all tests. I've had there 100, 50, 20 and now 4 during the development of the mock :)
I'll put 20 as default so this configuration would need to be changed rarely.

Reply:
I believe i'm aligned with @lemrey on this.

Why not simply set it to `100` or even more, so that it never has to be manually adjusted.

In case it's too low I assume you see weird run-time errors which can be hard for a developer to related to this setting, that is to know that to fix the error then this value must be increased.

I don't see a gain in saving memory in a unit test compared to risking hours of debugging weird errors.

Reply:
This value has been 20 for a while. I can certainly increase it to 100. However, if this is too small, you'll get a clear assertion with text:
"Too long string given to __wrap_custom_nrf_modem_at_scanf_ReturnVarg_string. Please increase CONFIG_MOCK_CUSTOM_NRF_MODEM_AT_SCANF_VARGS_STR_SIZE."